.github/workflows/community-releases.yml

```diff
@@ -0,0 +1,98 @@
+name: Docker Community Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to tag docker images'
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  packages: write
+  deployments: write
+  id-token: write
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    timeout-minutes: 80
+    steps:
+      - uses: actions/checkout@v4
+
+  docker_build:
+    runs-on: ubuntu-latest
+    needs: prepare
+    timeout-minutes: 80
+    environment: Production
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ['novu/api','novu/worker','novu/inbound-mail','novu/web','novu/webhook','novu/widget','novu/ws']
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Variables
+        shell: bash
+        run: |
+          service=${{ matrix.name }}
+          echo "SERVICE_NAME=$(basename "${service//-/-}")" >> $GITHUB_ENV
+          echo "REGISTRY_OWNER=novuhq" >> $GITHUB_ENV
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: 8.9.0
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.8.1
+          cache: 'pnpm'
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - uses: ./.github/actions/setup-qemu
+
+      - name: Login To Registry
+        shell: bash
+        env:
+          GH_ACTOR: ${{ github.actor }}
+          GH_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo $GH_PASSWORD | docker login ghcr.io -u $GH_ACTOR --password-stdin
+
+      - name: Build Community Docker Image with Buildx, tag, and test
+        shell: bash
+        env:
+          DOCKER_BUILD_ARGUMENTS: >
+            --cache-from type=registry,ref=ghcr.io/${{ env.REGISTRY_OWNER }}/cache:build-cache-${{ env.SERVICE_NAME }}-community
+            --cache-to type=registry,ref=ghcr.io/${{ env.REGISTRY_OWNER }}/cache:build-cache-${{ env.SERVICE_NAME }}-community,mode=max
+            --platform=linux/amd64,linux/arm64 --provenance=false
+            --output=type=image,name=ghcr.io/${{ env.REGISTRY_OWNER }}/${{ env.SERVICE_NAME }},push-by-digest=true,name-canonical=true
+        run: |
+          cd apps/$SERVICE_NAME && pnpm run docker:build
+          docker images
+
+      - name: Tag and Push docker image
```

Comment:
@AliaksandrRyzhou Should we also add a [health-check step](https://github.com/novuhq/novu/blob/f16076233ae6319c24989ec11b09ed8849ee48ee/.github/workflows/reusable-docker.yml#L162)?